lighthouse-core/audits/dobetterweb/js-libraries.js

```diff
@@ -57,9 +59,22 @@ class JsLibrariesAudit extends Audit {
     ];
     const details = Audit.makeTableDetails(headings, libDetails, {});
 
+    const debugData = {
+      type: /** @type {'debugdata'} */ ('debugdata'),
+      stacks: artifacts.Stacks.map(stack => {
+        /** @type {Record<string, string>} */
```

Comment:
this seems like a lot of gymnastics for `.map(s => ({id: s.id, version: s.version})` 🤔 

is this all to match whatever debugdata type is?

Reply:
i was just shaving some bytes from the json. `version` is normally undefined.

Reply:
`undefined` values are already pruned from the JSON by virtue of converting it to JSON :)

Reply:
you a smart :cookie: :)